### PR TITLE
Bump mono to 2017 06

### PR DESCRIPTION
**Pull request by luhenry**
cc @marek-safar

### CODE REVIEW

File: Make.config [hunk 1]
```diff
@@ -30,7 +30,7 @@ PACKAGE_HEAD_REV=$(shell git rev-parse HEAD)
 # on wrench, because wrench technically builds hashes, not branches)
 #
 # 
-PACKAGE_HEAD_BRANCH=master
+PACKAGE_HEAD_BRANCH=mono-2017-06
```
Comment (rolfbjarne) [inline Make.config]: This change must not be merged to master.

File: src/Security/Tls/AppleTlsProvider.cs [hunk 2]
```diff
@@ -21,23 +22,29 @@ namespace XamCore.Security.Tls
...
-			get { return id; }
+			get { return MNS.MonoTlsProviderFactory.AppleTlsId; }
 		}
 
 		public override IMonoSslStream CreateSslStream (
 			Stream innerStream, bool leaveInnerStreamOpen,
 			MonoTlsSettings settings = null)
 		{
-			return new AppleTlsStream (innerStream, leaveInnerStreamOpen, settings, this);
+			return SslStream.CreateMonoSslStream (innerStream, leaveInnerStreamOpen, this, settings);
```
Comment (spouliot) [inline src/Security/Tls/AppleTlsProvider.cs]: @baulig ^ can you confirm this is right ? thanks!

Comment (baulig) [reply src/Security/Tls/AppleTlsProvider.cs]: Yes, that's right.

**Comment by spouliot** [comment]
`[FAIL]	i18n 4/2 data files present: charinfo.nlp, collation.core.bin, collation.tailoring.bin, mscorlib.xml`

^ @marek-safar are those new, non `.bin` files known (and needed) additions in `mscorlib.dll` ?

File: Make.config (hunk 1 above)
Comment (luhenry) [reply Make.config]: Let me fix that

Comment (luhenry) [reply Make.config]: The value has been reverted to `master`

**Comment by spouliot** [comment]
@rolfbjarne the Mono.Security issue seems related to this morning discussion on slack, i.e. it should not be executed for watchOS (both on Jenkins and Wrench).

**Comment by rolfbjarne** [comment]
@spouliot yes, that’s https://bugzilla.xamarin.com/show_bug.cgi?id=57529, I’ll fix that in master (and d15-5 if needed).

**Comment by marek-safar** [comment]
@spouliot one of them is a new, where can I find the test so I can update it

**Pull request merged**